### PR TITLE
ENDOC-508 Update manage DB page with Notion

### PR DESCRIPTION
**Pull request by jyunmitch**

### CODE REVIEW

File: vuepress/docs/next/docs/reference/databases.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Entando can also be configured to use an existing
 DBMS provided by the customer. In these situations, lower level database
 operations such as tablespace creation, permissions and clustering must be carried out by the customer. 
 
-Entando then creates and populates the tables, indices and foreign keys. It creates these in the appropriate table 'container' for the DBMS, such
+Entando then creates and populates the tables, indices and foreign keys. It creates them in the appropriate table 'container' for the DBMS, such
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/reference/databases.md:52]: this reads: it creates and populates tables in the appropriate table named container; should remove backticks to make "table" an adj (table container) or remove table altogether for "appropriate `container` - that preserves backticking but doesn't precipitate some weird circular meaning. would condense and simplify so "it creates" doesn't apply to 2 consecutive sentences: "Entando then creates and populates the tables, indices and foreign keys in the appropriate 'container' for the DBMS..."

File: vuepress/docs/next/docs/reference/databases.md [hunk 2]
```diff
@@ -60,29 +60,34 @@ the same namespace as the EntandoApp and EntandoPlugin that use them. It is usua
...
-      name: string, any K8s compliant name
-      namespace: string, the namespace the DB is created in
-      stringData:
-        username: string, name of an admin user that can create schemas and other users
-        password: string, password of the above user
+``` yaml
+kind: "EntandoDatabaseService"
+apiVersion: "entando.org/v1alpha1"
+metadata:
+  name: string, any K8s compliant name
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/reference/databases.md:67]: K8s-compliant (work together to become one modifier, so hyphenated)

File: vuepress/docs/next/docs/reference/databases.md [hunk 3]
```diff
@@ -60,29 +60,34 @@ the same namespace as the EntandoApp and EntandoPlugin that use them. It is usua
...
+  namespace: string, the namespace this DB is created in
+spec:
+  dbms: string, one of Oracle, PostgreSQL or MySQL
+  host: string, either an IP address or hostname where the database service is hosted
+  port: integer, the port on which the database service is hosted
+  databaseName: string, the name of the database, only required for PostgreSQL and Oracle
+  secretName: the name of the Secret in the same namespace carrying admin credentials to the database service
+  tablespace: (Oracle only)  the tablespace to use for required schemas
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/reference/databases.md:75]: extra space before "the"

File: vuepress/docs/next/docs/reference/databases.md [hunk 4]
```diff
@@ -109,7 +114,7 @@ If the operator does not detect an
 to its default behaviour--creating a matching deployment and
 spinning up a database service from the same namespace. 
 
-If the `spec.dbms` property is not specified on an EntandoApp, the operator will
+If the `spec.dbms` property is not specified on an `EntandoApp`, the operator will
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/reference/databases.md:117]: an EntandoApp is an instance of an EntandoApp CR, and a property is specified *for* an instance, so "specified for"

File: vuepress/docs/next/docs/reference/databases.md [hunk 5]
```diff
@@ -294,52 +310,53 @@ To achieve this, specify the pertinent properties for the EntandoApp component i
...
+      value: admin
+    - name: SERVDB_PASSWORD
+      value: adminadmin
+    - name: SERVDB_CONNECTION_CHECKER
+      value: org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLValidConnectionChecker
+    - name: SERVDB_EXCEPTION_SORTER
+      value: org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLExceptionSorter
 ```
+**Note**: This configuration is not meant to be used as a template for a production environment.
+Use K8s Secrets in such cases to store the credentials and the syntax indicated here.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/reference/databases.md:352]: sentence is a little jumbled wrt actions associated with the verb "use" and "in such cases" can be avoided for a more direct and succinct instruction: "instead (as in for a production environment), store the credentials via K8s Secrets and use the syntax indicated here."

Comment (jyunmitch) [reply vuepress/docs/next/docs/reference/databases.md:352]: I don't think the instead works here. The first sentence is a general recommendation, and the second is one of the a specific details of note. I did edit it with part of the rec

Comment (Lyd1aCla1r3) [reply vuepress/docs/next/docs/reference/databases.md]: i misread it then - it took "in such cases" to mean that the second sentence was an extension of the first statement

**Review by Lyd1aCla1r3 — APPROVED** [review]

File: vuepress/docs/next/docs/reference/databases.md [hunk 6]
```diff
@@ -60,29 +60,34 @@ the same namespace as the EntandoApp and EntandoPlugin that use them. It is usua
...
-
-    Secret:
-      name: string, any K8s compliant name
-      namespace: string, the namespace the DB is created in
-      stringData:
-        username: string, name of an admin user that can create schemas and other users
-        password: string, password of the above user
+``` yaml
+kind: "EntandoDatabaseService"
+apiVersion: "entando.org/v1alpha1"
```
Comment (nshaw) [inline vuepress/docs/next/docs/reference/databases.md:65]: Change to apiVersion: entando.org/v1. I checked in PS Dev and that's the current apiVersion for the EntandoDatabaseService.

File: vuepress/docs/next/docs/reference/databases.md [hunk 7]
```diff
@@ -160,24 +164,28 @@ deployment operations will fail.
...
-          jdbcParameters: {}
-      Secret:
-        metadata:
-          name: oracle-secret
-        stringData:
-          username: admin
-          password: admin123
+``` yaml
+kind: "EntandoDatabaseService"
+apiVersion: "entando.org/v1alpha1"
```
Comment (nshaw) [inline vuepress/docs/next/docs/reference/databases.md:169]: -> apiVersion: entando.org/v1

File: vuepress/docs/next/docs/reference/databases.md [hunk 8]
```diff
@@ -219,23 +227,28 @@ higher.
...
-           useSSL: "true"
-    Secret:
-      metadata:
-        name: mysql-secret
-      stringData:
-        username: admin
-        password: admin123
+``` yaml
+kind: "EntandoDatabaseService"
+apiVersion: "entando.org/v1alpha1"
```
Comment (nshaw) [inline vuepress/docs/next/docs/reference/databases.md:232]: apiVersion: entando.org/v1

File: vuepress/docs/next/docs/reference/databases.md [hunk 9]
```diff
@@ -255,24 +268,27 @@ Entando Applications and plugins.
...
-
-      Secret:
-        metadata:
-          name: postgresql-secret
-        stringData:
-          username: admin
-          password: admin123
+``` yaml
+kind: "EntandoDatabaseService"
+apiVersion: "entando.org/v1alpha1"
```
Comment (nshaw) [inline vuepress/docs/next/docs/reference/databases.md:273]: apiVersion: entando.org/v1

**Pull request merged**